packages/retail-ui-extensions/src/extension-api/Models/Cart.ts

```diff
@@ -0,0 +1,55 @@
+export interface Cart {
+  subtotal: string;
+  taxTotal: string;
+  grandTotal: string;
+  note?: string;
+  cartDiscount?: Discount;
+  customer?: Customer;
+  lineItems: LineItem[];
+}
+
+export interface Customer {
+  id: number;
+  email?: string;
+  firstName?: string;
+  lastName?: string;
+  note?: string;
+}
+
+export interface LineItem {
+  price?: number;
+  quantity: number;
+  title?: string;
+  variantId?: number;
+  productId?: number;
+  discount?: Discount;
+  taxable: boolean;
+}
+
+export interface Discount {
+  amount: number;
+  discountDescription?: string;
+  type?: string;
+}
+
+export interface CustomSale {
+  quantity: number;
+  title: string;
+  price: string;
+}
+
+export interface Address {
+  address1?: string;
+  address2?: string;
+  city?: string;
+  company?: string;
+  firstName?: string;
+  lastName?: string;
+  phone?: string;
+  province?: string;
+  country?: string;
+  zip?: string;
+  name?: string;
+  provinceCode?: string;
+  countryCode?: string;
+}
```

Comment:
It would be great if we could try to get closer to the checkout shape for an address (https://github.com/Shopify/ui-extensions/blob/main/packages/checkout-ui-extensions/src/extension-points/api/standard/index.ts#L451-L463); a developer may never write both kinds of extensions but we are very clearly shipping the org chart with these subtly different shapes.